src/main/scala/com/github/pjfanning/sourcedist/SourceDistGenerate.scala

```diff
@@ -26,7 +27,8 @@ private[sourcedist] object SourceDistGenerate {
     ignoreList.addPatterns(customIgnorePatterns)
     val files = getIncludedFiles(baseDir, ignoreList)
 
-    val baseFileName  = s"$prefix-src-$version"
+    val versionString = if (incubating) s"$version-incubating" else version
```

Comment:
Extremely minor nit, but the variable name `versionString` here is slightly misleading because it also contains `incubating` (which has nothing to do with version).